frontend/src/components/navbar/Navbar.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import Brightness3OutlinedIcon from "@mui/icons-material/Brightness3Outlined";
+import LightModeOutlinedIcon from "@mui/icons-material/LightModeOutlined";
+import NavbarButtons from "../../utils/navbar-buttons";
+import { Link } from "react-router-dom";
+
+interface Props {
+  darkMode: boolean;
+  toggleDarkMode(): void;
+}
+
+const Navbar = ({ darkMode, toggleDarkMode }: Props) => {
```

Comment:
just a nit: would be better to name it NavbarProps, when this project become bigger, it would be easier to find which Props youre looking for, instead of having 20 Props from different files